lib/abiquo-chef-agent.rb

```diff
@@ -109,11 +109,11 @@ class Util
       #
       # Tries to find the right leases file
       #
-      def self.find_leases_file(search_dirs = ['/var/lib/dhcp3', '/var/lib/dhcp', '/var/lib/dhclient'])
+      def self.find_leases_file(search_dirs = ['/var/lib/dhcp3', '/var/lib/dhcp', '/var/lib/dhclient', '/var/lib/NetworkManager'])
         leases = []
         search_dirs.each do |d|
           Dir["#{d}/*"].each do |f|
-            next if f !~ /leases$/
+            next if f !~ /lease(s)?$/
```

Comment:
Add a 'dhclient.lease' file in [this directory](https://github.com/abiquo/abiquo-chef-agent/tree/master/spec/data/dhclient) and verify that the [util_spect.rb](https://github.com/abiquo/abiquo-chef-agent/blob/master/spec/unit/util_spec.rb) test still passes (you might have to update the `find_leases` test to accomodate the new behavior).